config/manager/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api-azure/cluster-api-azure-controller:latest
+      - image: gcr.io/cf-london-servces-k8s/nziada/cluster-api-azure-controller-amd64:dev
```

Comment:
oops... We need to stop manipulating files in place like this. It's too easy to check these in.

cc @CecileRobertMichon

Reply:
I actually tried to add this to gitignore last week and it didn't work (??) i don't know if I was doing it wrong or what kind of dark magic was going on...

Reply:
I always check this one in!